### PR TITLE
Forms Validation.md

### PR DESCRIPTION
**Pull request by medhaanand**

### CODE REVIEW

File: docs/bot-builder/basic/forms.md [hunk 1]
```diff
@@ -117,6 +115,53 @@ To create this picker form field just select the type as **Picker** and in the O
...
+
+2. For **all other fields**: For fields apart from **text**, there is *Input Validations*.
+
+![Form15](assets/Form15.png)
+
+This has only two values *Optional* and *Mandatory*.
+
+![Form16](assets/Form16.png)
+
+Every validation can go wrong. To inform the user that he has given an incorrect input, we can configure **Error Message**. This is right below the validation fields.
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/forms.md:157]: Can we make this gender neutral `To inform the user that he has given`?

File: docs/bot-builder/basic/forms.md [hunk 2]
```diff
@@ -117,6 +115,53 @@ To create this picker form field just select the type as **Picker** and in the O
...
+
+![Form17](assets/Form17.png)
+
+If the user inputs incorrect values defying the validations, he will get to see the error message as shown below -
+
+![Form18](assets/FormError.png)
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/forms.md:163]: The screenshot has many fields with validations without the error message. Can we change this screenshot?

File: docs/bot-builder/basic/forms.md [hunk 3]
```diff
@@ -117,6 +115,53 @@ To create this picker form field just select the type as **Picker** and in the O
...
+
+This has only two values *Optional* and *Mandatory*.
+
+![Form16](assets/Form16.png)
+
+Every validation can go wrong. To inform the user that he has given an incorrect input, we can configure **Error Message**. This is right below the validation fields.
+
+![Form17](assets/Form17.png)
+
+If the user inputs incorrect values defying the validations, he will get to see the error message as shown below -
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/forms.md:161]: Can we make this statement gender neutral `validations, he will get to see`?

Comment (tejas-haptik) [reply docs/bot-builder/basic/forms.md:161]: Or does our product adds validations only for men to keep them in check?

Comment (medhaanand) [reply docs/bot-builder/basic/forms.md:161]: ha ha very funny...

File: docs/bot-builder/basic/forms.md (hunk 2 above)
Comment (medhaanand) [reply docs/bot-builder/basic/forms.md:163]: Done

**Review by tejas-haptik — APPROVED** [review]
Looks good

**Pull request merged**